Flow.Launcher.Plugin.Everything.csproj

```diff
@@ -84,20 +68,25 @@
       <AutoGen>True</AutoGen>
       <DependentUpon>Settings.settings</DependentUpon>
     </Compile>
+    <Compile Remove="Output\**" />
```

Comment:
think there is a better way to do this right instead of add this to here, how's flow doing it?

Reply:
Hmmm I am not sure, will take a look once I get back to home.

Reply:
all good no stress, if i have time in front of comp i will do it plus remove the launchSettings.json file